docs/sections/workflows/writing_basics.md

```diff
@@ -175,103 +154,445 @@ See the {ref}`corresponding topics section for more details <topics:processes:us
+
+
+    def add(self):
+        """Add two integers."""
+
+        # Call `add` using a variable from the context and one of the inputs
+        result = add( self.ctx.product, self.inputs.z )
+
+        # Parsing the output
+        self.out("final_result", result)
+```
+
+In `add()`, we called the `addition()` calculation function (declared outside of the class) using a variable from the context and one of the inputs.
+Then, we declared the last output of the work chain.
+And that is it.
+We finished our very first and simple work chain.
+
+Let's run it!
+In the terminal, navigate to the folder where you have your work chain file.
+Then, open a `verdi shell` session and execute:
+
+
+```{code-block} ipython
+In [1]: from aiida.engine import run
+In [2]: from multiply_add_workchain import MultiplyAddWorkChain
+In [3]: result = run(MultiplyAddWorkChain, Int(2), Int(3), Int(5) )
+In [4]: result
+Out[2]:
+{'product': <Int: uuid: 9c5c29de-6176-41fe-a051-672a5348e631 (pk: 1909) value: 6>,
+ 'final_result': <Int: uuid: 3dbebeb3-7ed4-4331-976c-94099a305622 (pk: 1911) value: 11>}
+```
+Check the list with last processes:
+
+```{code-block} console
+$ verdi process list -a -p 1
+...
+1899  7s ago     MultiplyAddWorkChain  ⏹ Finished [0]
+1900  7s ago     multiplication        ⏹ Finished [0]
+1902  7s ago     add                   ⏹ Finished [0]
+
+```
+Also check the status of the process that corresponds to the work chain `MultiplyAddWorkChain`:
+```{code-block} console
+$ verdi process process status 1899
+MultiplyAddWorkChain<1899> Finished [0] [1:add]
+    ├── multiplication<1900> Finished [0]
+    └── add<1902> Finished [0]
+```
+And show some details about the inputs and outputs with:
+```{code-block} console
+$ verdi process show 1899
+Property     Value
+-----------  ------------------------------------
+type         MultiplyAddWorkChain
+state        Finished [0]
+pk           1899
+uuid         d15bff43-1e03-4e21-adc3-6065e6bd205f
+label
+description
+ctime        2021-06-08 13:37:12.256669+00:00
+mtime        2021-06-08 13:37:13.260633+00:00
+
+Inputs      PK  Type
+--------  ----  ------
+x         1896  Int
+y         1897  Int
+z         1898  Int
+
+Outputs         PK  Type
+------------  ----  ------
+final_result  1903  Int
+product       1901  Int
+
+Called      PK  Type
+--------  ----  --------------
+CALL      1900  multiplication
+CALL      1902  add
+```
+
+:::{note}
+There are other ways to run/submit your work chain. ADD LINK.
+:::
+
+#### Work chain with Calculation Jobs
+
+Here, we demonstrate how to include calculation jobs in our work chain.
+Calculation jobs represent external codes in AiiDA.
+We are going to use the example in Section {ref}`Constructing our first work chain<Constructing our first work chain>` and replace the addition calculation function:
+
+```{literalinclude} include/code/new_scripts/multiply_add_workchain_calcjob.py
+:language: python
+```
+
+Let us break down what we did.
```

Comment:
If you agree with the above comment, this needs to be changed somewhat. But maybe we should first agree on that. ;)